website/docs/components/inline-edit/inline-edit-code.md

```diff
@@ -5,7 +5,7 @@ tabs: Design('inline-edit'), A11y('inline-edit-a11y'), API('inline-edit-api'), E
 
 ## Basic usage
 
-You can use `<InlineEdit />` to make simple text editable.
+Use `<InlineEdit />` to make simple text wrapped in [InlineInput](/components/inline-input/inline-input) editable.
```

Comment:
```suggestion
Use `<InlineEdit />` to make plain text wrapped in [InlineInput](/components/inline-input/inline-input) editable.
```
I think "plain text" is a more suitable term) and vale will stop complaining about "simple"

Reply:
Yeah! Wasn't sure, because have seen both variants. Thank you ✨